pkg/policy/api/selector.go

```diff
@@ -158,6 +159,7 @@ func (n EndpointSelector) GetMatch(key string) ([]string, bool) {
 func labelSelectorToRequirements(labelSelector *metav1.LabelSelector) *k8sLbls.Requirements {
 	selector, err := metav1.LabelSelectorAsSelector(labelSelector)
 	if err != nil {
+		metrics.PolicyImportErrors.Inc()
```

Comment:
We should ideally never hit errors here and should resolve this at import time - I filed  https://github.com/cilium/cilium/issues/7176 to account for that. Having the metric here in the meantime is good.